src/test/groovy/org/gradle/android/TestVersions.groovy

```diff
@@ -40,7 +40,7 @@ class TestVersions {
         return minorVersions.collect { getLatestVersionForAndroid(it) }
     }
 
-    static List<String> supportedKotlinVersions = ["1.5.32", "1.6.10"]
+    static List<String> supportedKotlinVersions = ["1.6.10"]
```

Comment:
should we add 1.7.22 here or is that just a completely independent PR (maybe there will be merge conflicts, let's make sure it ends up being correct)

Reply:
yep, next pr includes de 1.7  update